downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -202,7 +197,7 @@ abstract class VersionControlSystem {
     fun isApplicableUrl(vcsUrl: String): Boolean {
         if (vcsUrl.isBlank() || vcsUrl.endsWith(".html")) return false
 
-        return isApplicableType(VcsHost.parseUrl(vcsUrl).type) || isApplicableUrlInternal(vcsUrl)
+        return type == VcsHost.parseUrl(vcsUrl).type || isApplicableUrlInternal(vcsUrl)
```

Comment:
The removal of the public `isApplicableType` function should make this a breaking change.

Reply:
Yeah, I was a bit torn between keeping `isApplicableType()` (and using it more consistently), or removing it as it's so trivial. Any preference?

Reply:
Not really, we don't use it in the server code either, so I guess it's fine to remove it.